api/v1/views/index.py

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/python3
+"""
+    index.py file in v1/views
+"""
+from flask import Flask, jsonify
+from api.v1.views import app_views
```

Comment:
move this line to line 5 so our imports are alphabetical

Reply:
Will update in latest pull